frontend/src/components/ui-kit/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import styles from './Checkbox.module.scss';
+
+const Checkbox = ({ label, checked, onChange }) => {
+  return (
+    <label className={styles.container}>
+      <input type="checkbox" checked={checked} onChange={onChange} />
+      {label}
+    </label>
+  );
+};
+
+const FlowerPicker = () => {
+  const types = [
```

Comment:
Put all these constants behind the component, or create a folder for constants (see project structure)